### PR TITLE
Monthly modal updates

### PR DESCRIPTION
**Pull request by flukeout**
Some minor styling updates to the monthly donation ask modal. 

* Slightly different treatment of the currency input
* Tweaked modal width, font-sizes, etc. depending on screen width

Should look like...

image

### CODE REVIEW

File: less/components/monthly-upgrade.less [hunk 1]
```diff
@@ -1,35 +1,73 @@
 .upgrade-container {
   .modal-container {
     background: #D8D5CC;
+    overflow: hidden;
   }
   .upgrade-modal {
     background: #FFFFFF;
-    padding: 0 30px 30px 30px;
+    padding: 30px 30px 30px 30px;
```
Comment (gideonthomas) [inline less/components/monthly-upgrade.less]: nit: `padding: 30px;`

Comment (flukeout) [reply less/components/monthly-upgrade.less]: Fixing

**Comment by flukeout** [comment]
@WillatMozFdn You can view this modal directly on the deploy environment via [this link](https://donate-mozilla-org-us--pr-1921.herokuapp.com/en-US/thank-you/?payment=Stripe&str_amount=5000&str_currency=usd&str_id=ch_1BRlDKG8Mmx3htnxif3bcq5I&str_frequency=one-time&email=maryroe@example.org&country=Canada).

**Comment by WillatMozFdn** [comment]
@flukeout -- I'm getting a "No Such App" message on Heroku when I try to test it out at  https://donate-mozilla-org-us--pr-1921.herokuapp.com/ ... ("There's nothing here yet ... build something amazing.") Did I miss something over the weekend as it was merged?  Can you send me another link to test?  Thanks!

**Comment by ScottDowne** [comment]
It's merged now so staging at the least.

**Comment by flukeout** [comment]
@WillatMozFdn here's a [direct link](https://donate.mofostaging.net/en-US/thank-you/?payment=Stripe&str_amount=5000&str_currency=usd&str_id=ch_1BRlDKG8Mmx3htnxif3bcq5I&str_frequency=one-time&email=maryroe@example.org&country=Canada) to the modal on staging.

**Pull request merged**